### PR TITLE
Migration from PlanSys2 organization

### PR DESCRIPTION
**Pull request by fmrico**
Dear mantainers,

This is the migration fo the repo in PlanSys2 organization to aiplan4eu organization, except plansys2_upf_plan_solver, which continues in PlanSys2.

Best
Francisco

### CODE REVIEW

File: docs/README.md
Comment (g-gemignani) [inline docs/README.md:1]: @fmrico, thank you so much for the PR! Shouldn't the entire plansys2_upf folder be removed from this pr?

Comment (fmrico) [reply docs/README.md:1]: Yes!!. I will remove it and keep it in the PlanSys2 repo

Comment (fmrico) [reply docs/README.md:1]: Wait for a new commit ;)

**Comment by mikand** [comment]
@fmrico @g-gemignani Thanks for this! Can we merge?

**Comment by g-gemignani** [comment]
@mikand this PR is currently broken. I took over @fmrico's work yesterday and I want to get it to work properly soonish. I'll have an update in the next few weeks. Is this urgent?

**Comment by mikand** [comment]
@g-gemignani No urgency: I just noticed that the master branche is empty :)

**Comment by g-gemignani** [comment]
@mikand I'll try to fix it asap :)

**Comment by mikand** [comment]
:heart:

**Comment by fmrico** [comment]
Thanks @mikand @g-gemignani :nerd_face:

**Pull request merged**